pelican/contents.py

```diff
@@ -4,10 +4,15 @@
 import logging
 import os
 import re
+from datetime import timezone
 from html import unescape
 from urllib.parse import unquote, urljoin, urlparse, urlunparse
 
-import pytz
+try:
+    import zoneinfo
+except ModuleNotFoundError:
+    import backports.zoneinfo
+
```

Comment:
I'd suggest doing the import like this:
```python
try:
    import zoneinfo
except ModuleNotFoundError:
    from backports import zoneinfo
```

Then you would simplify things like:
```python
        try:
            self.timezone = zoneinfo.ZoneInfo(timezone)
        except NameError:
            self.timezone = backports.zoneinfo.ZoneInfo(timezone)
```
with this
```python
        self.timezone = zoneinfo.ZoneInfo(timezone)
```

Reply:
Can you make this same improvement in the other two relevant files in this PR? That is:
- `pelican/tools/pelican_quickstart.py`
- `pelican/utils.py`